registry.k8s.io/images/k8s-staging-kas-network-proxy/images.yaml

```diff
@@ -114,6 +115,7 @@
     "sha256:5da486beb32f397004ccae9af53618ca4db527a2a33445fa20ec8d94ac5bbf5d": ["v0.29.1"]
     "sha256:798b44167c71dd2a1388c3a36920f20aac062545c9518fb807ccae81c015b95c": ["v0.29.2"]
     "sha256:a16e56961c8e417d487666446d081e6fce691063f0530834c0df53b6b3121cef": ["v0.29.3"]
+    "sha256:a16e56961c8e417d487666446d081e6fce691063f0530834c0df53b6b3121cef": ["v0.29.4"]
```

Comment:
Due to infrastructure issues at ANP, 0.29.3 is not built correctly, https://github.com/kubernetes-sigs/apiserver-network-proxy/tree/v0.29.3

But maybe it's not a big deal
@jkh52

Reply:
oops, seems like this is not allow to same key here, so i delete the tag of 0.29.3
```
time="07:34:13.727" level=fatal msg="run `cip run`: promote images: parsing manifests: parsing thin manifest directory: yaml: unmarshal errors:\n  line 56: key \"sha256:e5d4082225403e2c46ddcd55a951321fa8ed8922e458cc4b4d88000a6e240f99\" already set in map\n  line 118: key \"sha256:a16e56961c8e417d487666446d081e6fce691063f0530834c0df53b6b3121cef\" already set in map" diff=0s

```

Reply:
I think we can't / shouldn't delete an existing image, once it has been published. @BenTheElder could you help advise or suggest someone?

Reply:
> I think we can't / shouldn't delete an existing image, once it has been published. @BenTheElder could you help advise or suggest someone?

Yeah, you can't mutate image tags on registry.k8s.io
It's intended to protect users, I guess. The policy has been around for a long time now.
Ideally it would give a nicer warning in presubmit, I think I filed an issue about this last year.

Reply:
> It's intended to protect users, I guess. The policy has been around for a long time now.

(Also removing / mutating images has some annoying logistical properties anyhow, since we're replicating it across providers and locations)